other_processing/0.download-profiles-from-figshare/download.py

```diff
@@ -0,0 +1,88 @@
+"""
+Download Single Cell Cell Painting Profiles
+Gregory Way, 2019
+
+These data were output from a CRISPR and Cell Painting experiment.
+A microscope took pictures of all cells under several CRISPR perturbations.
+The data were processed by a custom CellProfiler pipeline.
+In the pipeline, we measured several morphology features for every single cell.
+We then aggregated the output of CellProfiler using cytominer-database.
+Cytominer-database compiles all single cell profiles into a single `sqlite` database.
+We uploaded the `sqlite` files to NIH Figshare 
+https://nih.figshare.com/articles/dataset/Cell_Health_-_Cell_Painting_Single_Cell_Profiles/9995672/1.
+These data are publicly available
+
+*Important Note*
+
+These files are large (~130 GB Total).
+There is no need to download and reprocess the data.
```

Comment:
can you clarify if this is still the case?

Reply:
also, please add details about _why_ we're downloading.

It is to convert to `parquet` files and create version 6 of the data, correct?

Reply:
> can you clarify if this is still the case?
In terms of downloading, yes. Pipeline requires to download the raw dataset, which is 130GB. 

changes are applied on: 4ada9d2258d8f412b369ccdf4c47c9212278eff5